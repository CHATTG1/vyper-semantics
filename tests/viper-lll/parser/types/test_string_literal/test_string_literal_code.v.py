@public
def foo() -> bytes <= 5:
    return "horse"
