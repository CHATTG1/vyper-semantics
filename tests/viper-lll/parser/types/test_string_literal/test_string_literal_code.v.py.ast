%pgm(, ,
  %fdecl(%@public, foo, , %bytesT(5),
    %return("horse"))
)
