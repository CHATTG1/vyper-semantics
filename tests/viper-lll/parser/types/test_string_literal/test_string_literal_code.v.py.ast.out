$%seq ( $return ( 0 , $lll ( $%seq ( $%seq ( $mstore ( 28 , $calldataload ( 0 ) ) , $mstore ( 32 , 1461501637330902918203684832716283019655932542976 ) , $mstore ( 64 , 170141183460469231731687303715884105727 ) , $mstore ( 96 , -170141183460469231731687303715884105728 ) , $mstore ( 128 , 1701411834604692317316873037158841057270000000000 ) , $mstore ( 160 , -1701411834604692317316873037158841057280000000000 ) ) , $if ( $eq ( $mload ( 0 ) , 3264763256 ) , $%seq ( $pass , $assert ( $iszero ( $callvalue ) ) , $with ( $_loc , $%seq ( $mstore ( 320 , 5 ) , $mstore ( $add ( 320 , 32 ) , 47237446186964047356529078487726681904857602541872513497069725882920290222080 ) , 320 ) , $%seq ( $mstore ( $sub ( $_loc , 32 ) , 32 ) , $return ( $sub ( $_loc , 32 ) , $ceil32 ( $add ( $mload ( $_loc ) , 64 ) ) ) ) ) , $stop ) ) ) , 0 ) ) )
